frontstage/views/surveys/add_survey_submit.py

```diff
@@ -49,4 +59,12 @@ def add_survey_submit(session):
     logger.info('Successfully retrieved data for confirm add organisation/survey page',
                 case_id=case_id, party_id=party_id)
     return redirect(url_for('surveys_bp.get_survey_list', _anchor=(business_party_id, added_survey_id), _external=True, business_party_id=business_party_id,
-                            survey_id=added_survey_id, tag='todo'))
+                            survey_id=added_survey_id, tag='todo', already_enrolled=already_enrolled))
+
+
+def is_business_enrolled(associations, survey_id):
+    for info in associations:
```

Comment:
This can probably be condensed down to an `any` statement.

```suggestion
    return any(survey for info in associations for survey in info['enrolments'] if survey['surveyId'] == survey_id)
```

Reply:
I like that, just tested it and made the change.